ui/src/utils/constants.ts

```diff
@@ -107,3 +107,59 @@ export const SCORE_THRESHOLDS = {
   WARNING: 0.8,
   ALERT: 0.6,
 }
+
+/**
+ * Calculate dynamic score thresholds based on a project's score threshold.
+ * 
+ * This creates relative ranges that provide meaningful visual feedback:
+ * - Red: Any score below the project threshold OR in the lower 10% above it
+ * - Orange: Middle range (10% to 60% of the range above project threshold)  
+ * - Green: Upper range (above 60% of the range above project threshold)
+ * 
+ * @param projectScoreThreshold - The project's configured score threshold (e.g., 0.6)
+ * @returns Object with ALERT and WARNING thresholds for use in styling logic
+ * 
+ * @example
+ * // With project threshold of 0.6:
+ * const thresholds = calculateDynamicScoreThresholds(0.6)
+ * // Returns: { ALERT: 0.64, WARNING: 0.84 }
+ * // Red: < 0.64, Orange: 0.64-0.84, Green: >= 0.84
+ */
+export const calculateDynamicScoreThresholds = (
```

Comment:
I gave the dynamic scores some thought. It's a cool experiment, but I think I suggest we keep the thresholds constant, for explainability reasons 🤔 However in future, it would be nice to make them configurable for the project, like the filter threshold.